### PR TITLE
Added Alert Bar component

### PR DESCRIPTION
**Pull request by ShreyaThapa**

### CODE REVIEW

File: components/base/AlertBar.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import {
+  GREY_1,
+  GREY_2,
+  API_STATUS_BAR_YELLOW,
+  PARAGRAPH_TEXT,
+} from "../colors";
+import { ROBOTO } from "../typography";
+
+const StyledAlertBar = styled.div`
+  display: flex;
```
Comment (Finholt) [inline components/base/AlertBar.tsx:14]: What are your thoughts on adding `align-items: center;` here in order to vertically center the icon if the text wraps to more than one line? I'm going back and forth

Comment (ShreyaThapa) [reply components/base/AlertBar.tsx:14]: Yeah, I was going back and forth on that too! 

I tried going off of this [page](https://projects.invisionapp.com/d/main#/console/19172953/400928381/preview) in the mock-ups that has the icon positioned at the top when the text wraps.  Thought it looked better that way too. Let's ask the team about it. 👍

Comment (ShreyaThapa) [reply components/base/AlertBar.tsx:14]: Looks like the verdict is to keep it positioned at the top. 🔝

Comment (Finholt) [reply components/base/AlertBar.tsx:14]: Sounds good! Thanks for talking through that with me/the team 💯

**Review by Finholt — APPROVED** [review]

File: components/base/AlertBar.tsx (hunk 1 above)
Comment (ShreyaThapa) [reply components/base/AlertBar.tsx:14]: 😎👍

**Pull request merged**